ui/admin/app/styles/app.scss

```diff
@@ -94,6 +94,8 @@
 .rose-table {
   p {
```

Comment:
To reduce accidentally applying this style to all table columns, could this be restricted to just the header cells in a table?

Reply:
good catch. updated!